tests/nnstreamer_datarepo/unittest_datareposrc.cc

```diff
@@ -45,8 +45,10 @@ bus_callback (GstBus *bus, GstMessage *message, gpointer data)
 {
   switch (GST_MESSAGE_TYPE (message)) {
     case GST_MESSAGE_EOS:
+    case GST_MESSAGE_ERROR:
       g_main_loop_quit ((GMainLoop *) data);
       break;
+      break;
```

Comment:
unnecessary?

Reply:
When I do git rebase, the merge seems to be wrong.
Thank you.